### PR TITLE
56781: Updated Kubernetes Version Compatibility Table

### PR DESCRIPTION
**Pull request by jonquil2002**
SC: https://app.shortcut.com/replicated/story/56781/kubernetes-version-compatibility-table-should-not-include-unsupported-k8s-version

Preview: https://deploy-preview-597--replicated-docs.netlify.app/enterprise/installing-general-requirements#kubernetes-version-compatibility

### CODE REVIEW

File: docs/enterprise/installing-general-requirements.md [hunk 1]
```diff
@@ -21,25 +21,25 @@ The following table lists the browser requirements for the latest Replicated adm
 
 ## Kubernetes Version Compatibility
 
-Each release of the open source KOTS project maintains compatibility with the current Kubernetes version, and the two most recent versions at the time of its release.
+Each release of the open source KOTS project maintains compatibility with the current Kubernetes version, and the two most recent versions at the time of its release. This includes support against all patch releases of the corresponding Kubernetes version.
 
-This includes support against all patch releases of the corresponding Kubernetes version.
+Kubernetes versions 1.21 and earlier are end-of-life (EOL), however Replicated still maintains KOTS support for the EOL Kubernetes versions listed in the following table. For more information about Kubernetes versions, see [Release History](https://kubernetes.io/releases/) in the Kubernetes documentation.
```
Comment (JustinN-Replicated) [inline docs/enterprise/installing-general-requirements.md:26]: I think it is better for us to reference their cadence vs a specific version. Here is something that could be considered as an alternative:

>**NOTE:** Kubernetes has an N-2 support policy. This means that the three most recent versions are actively supported. The Kubernetes [release cadence](https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/2572-release-cadence) specifies 3 releases a year, each with a 15 week duration. Replicated maintains KOTS support with the Kubernetes versions listed in the following table. For more information about Kubernetes versions, see [Release History](https://kubernetes.io/releases/) in the Kubernetes documentation.

Comment (AmberAlston) [reply docs/enterprise/installing-general-requirements.md:26]: As-is this introduces language the implies a support policy statement that we don't want to make here. Will tweak it further.

**Review by AmberAlston — CHANGES_REQUESTED** [review]
Good with this with my edit incorporated - https://github.com/replicatedhq/replicated-docs/pull/597/commits/c330d836f5b6a1a05c5e537e2fb25790709477e0

**Comment by jonquil2002** [comment]
@AmberAlston @JustinN-Replicated Awesome. Just made a minor wording change for style. Please do a final review/approval, then I'll ask Paige to review/merge. Thanks!

**Review by AmberAlston — APPROVED** [review]

**Review by paigecalvert — APPROVED** [review]
Thanks @jonquil2002 ! Looks great. This could be part of another story, but curious, did you discuss with product at all the possibility of adding a warning about the unsupported versions of Kubernetes?

**Comment by jonquil2002** [comment]
@paigecalvert  We did not discuss a warning per se, but they are actively getting vendors to migrate to newer versions and still supporting older versions via patches. They do not want to give the impression that we're not supporting older versions when in fact we still are. They're not ready for a more formal statement yet, so I'm thinking maybe we do a separate story when they are ready, perhaps with a warning. They did not want to eliminate anything from the table either, at this time.

**Pull request merged**